src/library-authoring/LibraryContent.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect } from 'react';
```

Comment:
Replaced LibraryHome, LibraryCollections and Library Components with this single file that does the same based on the `content` prop.